filters/builtin/path.go

```diff
@@ -79,13 +89,36 @@ func createSetPath(config []interface{}) (filters.Filter, error) {
 	return &modPath{behavior: fullReplace, replacement: replacement}, nil
 }
 
+func createTransformPath(config []interface{}) (filters.Filter, error) {
+	if len(config) != 1 {
+		return nil, filters.ErrInvalidFilterParameters
+	}
+
+	replacement, ok := config[0].(string)
+	if !ok {
+		return nil, filters.ErrInvalidFilterParameters
+	}
+
+	rx := regexp.MustCompile("\\$\\{(\\w+)\\}")
+	matches := rx.FindAllStringSubmatch(replacement, -1)
+	placeholders := make([]string, len(matches))
+
+	for index, placeholder := range matches {
+		placeholders[index] = placeholder[1]
```

Comment:
I am not sure if `FindStringSubmatch` is enough, it will only find the leftmost one, I want to find all `${...}` maybe there are more than one.